app/views/posts/index.html.erb

```diff
@@ -1 +1,18 @@
-<h1>list of posts for a given user </h1>
\ No newline at end of file
+<h1>list of posts for a given user </h1>
+<%= render 'shared/user', user:@user%>
+ <div class="post_list">
+   <% @posts.each do |post| %>
```

Comment:
- [ ] I got an error when I click the button "See all posts", kindly fix the error and submit for a review.

![posts](https://github.com/alicemirigo92/blog_post/assets/53124548/740318f0-16ac-4540-ab3b-7faa3e6f914f)

Reply:
- [ ] [REQUIRED] Good job @alicemirigo92 fixing the **See all Post** button and making sure it opens the user's post index, however, you need to fix the UI of the user's post index page to look like the wireframe provided.

### Your Design
<img width="1434" alt="Screenshot 2023-05-18 at 21 18 00" src="https://github.com/alicemirigo92/blog_post/assets/58692590/23d1a352-7d9c-4464-b3c2-17a80cf82cd0">
### Wireframe Provided
<img width="508" alt="Screenshot 2023-05-18 at 21 17 30" src="https://github.com/alicemirigo92/blog_post/assets/58692590/b4e82ba0-9ac5-438b-a8d1-7c11554360f9">